src/qml/qgismobileapp.qml

```diff
@@ -59,6 +59,8 @@ ApplicationWindow {
     Component.onCompleted: focusstack.addFocusTaker( this )
   }
 
+  property Rubberband currentRubberband
```

Comment:
Can you add a docstring explaining that this is used as a proxy to the digitizing or the measuring rubberband, I think this will help us remember in the future :)